.env.sample

```diff
@@ -4,7 +4,13 @@
 CLIENT_ID=<client id>
 CLIENT_SECRET=<client secret>
 
-# Expects a full authority URL such as "https://login.microsoftonline.com/TENANT_GUID"
-# or "https://login.microsoftonline.com/contoso.onmicrosoft.com".
+# The AUTHORITY variable expects a full authority URL.
+#
+# If you are using an AAD tenent, configure it as
+# "https://login.microsoftonline.com/TENANT_GUID"
+# or "https://login.microsoftonline.com/subdomain.onmicrosoft.com".
+#
+# If you are using a CIAM tenant, configure it as "https://subdomain.ciamlogin.com"
```

Comment:
One concern I have with having so much information inside the code sample is internationalization- I assume that this will still show up as English when embedded in non-en translated sites, unless they have some approach to translate it. I don't know if there's a single place to put this info on learn.com, however, so maybe this is the best approach of the options.

Reply:
I understand where you are coming from, and it is true all the comments in this `env.sample` could show up as-is when embedded in non-en translated sites *(more on this in a minute)*. I think that is fine in a sense that the programming source code is historically English-dominant, and this `env.sample` configuration file is technically still part of the source code.

I would still suggest to keep these comments here as the single source of truth which will more likely stay in-sync with the underlying sample logic, simply because they are in the same repo.

Note that the English comments inside this configuration file still allows the content team to reference line 16 `#AUTHORITY=<authority url>` ONLY, while maintaining multi-language instructions on learn.microsoft.com, should they choose to do so.

So, now you can decide whether to go with the aforementioned "referencing line 16 only" approach, or to render the entire `env.sample` as-is (so that you do not have to update [the quickstart doc](https://learn.microsoft.com/en-us/azure/active-directory/develop/web-app-quickstart?pivots=devlang-python&tabs=windows#step-5-configure-the-sample-app) as often).